example/SeqLog4NetExample/App.config

```diff
@@ -1,38 +1,38 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Seems like only whitespace changes in this file - can we revert to keep the history clear?